beacon_chain/types/src/deposit.rs

```diff
@@ -10,6 +11,29 @@ pub struct Deposit {
     pub deposit_data: DepositData,
 }
 
+impl Deposit {
+    pub fn zero_with_rand_keypair() -> Self{
```

Comment:
Sorry I hadn't communicated this, but I'd like to remove the `zero_with_rand_keypair()` functions in favor of the `TestRandom` trait.

I had a quick look and it should be pretty easy to swap out.

Reply:
That's much cleaner.

Reply:
I'll remove the one in `ValidatorRecord` as well